app/email_branding/rest.py

```diff
@@ -18,6 +19,18 @@
 register_errors(email_branding_blueprint)
 
 
+@email_branding_blueprint.errorhandler(IntegrityError)
+def handle_integrity_error(exc):
+    """
+    Handle integrity errors caused by the unique constraint on uq_email_branding_name
+    """
+    if "uq_email_branding_name" in str(exc):
+        return jsonify(result="error", message="Email branding name already exists"), 400
```

Comment:
Given this is api, this returns in eng. I think its fine as its the admin page, but if there is a way to add to the admin page so its fr as well, that would be idea.

Reply:
We can pass the english string from the API to admin and translate it in admin, yes.